frontend/src/metabase/components/ArchivedItem/ArchivedItem.tsx

```diff
@@ -56,7 +56,7 @@ export const ArchivedItem = ({
     {isAdmin && (onUnarchive || onDelete) && (
       <span className="ml-auto mr2">
         {onUnarchive && (
-          <Tooltip tooltip={t`Unarchive this ${type}`}>
+          <Tooltip tooltip={t`Unarchive this ${model}`}>
```

Comment:
The underlying model reveals the item as we call it in the code, but we want its colloquially used name instead.

E.g.
`model` instead of `dataset`
`question` instead of `card`

![image](https://github.com/metabase/metabase/assets/31325167/08ca9449-7c9d-4928-91ee-c421e260ea4e)

![image](https://github.com/metabase/metabase/assets/31325167/0d24fc48-6e80-417e-aed5-b92fbb0c6a43)

@alxnddr where would be a good place to apply this logic? Should I create a helper that takes in `model` and spits out the `type` before I use it in the tooltip?

Also, if we have a fairly limited set of item types we can archive, does it make sense to narrow down the types for `model` from `string` to a union type of specific strings?

Reply:
We should keep it translated so let's use `getTranslatedEntityName` from `frontend/src/metabase/nav/utils/model-names.ts` which should be moved out of `nav` module in this case. Limiting model types would be great too. We need to define a union type in the `metabase-types/api` and use it instead of strings in related API types. Then we can use it in `frontend/src/metabase/nav/utils/model-names.ts`

Reply:
Let me know if you want to handle it by yourself or you need any help!

Reply:
I realized there is a `CollectionItemModel` type and have used it in https://github.com/metabase/metabase/pull/33970/commits/28c6e38576f20c729f42e09a6e617c129021f01a.

Would that be enough or would you rather narrow it down even further?
I have to double check what can be archived.

Let me learn a bit more about `getTranslatedEntityName`. Thanks for the hint.

Reply:
Nice, we already have `CollectionItemModel` so you can use it in `frontend/src/metabase/nav/utils/model-names.ts`. I would not narrow down it any further